everviz/config.py

```diff
@@ -1,24 +1,60 @@
 import os
 import yaml
+from collections import namedtuple
 from everviz.log import get_logger
+from everviz.pages import controls
 
+DataSources = namedtuple("DataSource", "controls_per_batch, controls_initial_vs_best")
 
-def _create_folders(file_path):
-    folder_path = os.path.realpath(os.path.dirname(file_path))
-    if not os.path.isdir(folder_path):
-        os.makedirs(folder_path)
+logger = get_logger()
 
 
-def webviz_config():
+def _get_everviz_folder(everest_folder):
+    everviz_path = os.path.join(everest_folder, "everviz")
+    if not os.path.exists(everviz_path):
+        os.makedirs(everviz_path)
+    return everviz_path
+
+
+def _write_to_csv(data, file_path, sep=","):
+    data.to_csv(file_path, sep=sep, index=False)
+    logger.info("File created: {}".format(file_path))
+    return file_path
+
+
+def _set_up_data_sources(api):
```

Comment:
Could this function be moved into the controls module? Maybe the modules that make the pages, would only have to expose then a single function, like `controls.page_layout(api)`. To add a new page, only a new import, and one call to that modules layout function would have to be added.

Reply:
I can do that, and see how that works exactly and if it gets too big for a function we can refactor it at a later date